tensordict/tensordict.py

```diff
@@ -1424,6 +1424,36 @@ def reshape(
             batch_size = shape
         return TensorDict(d, batch_size, device=self.device)
 
+    def split(
+        self, split_size: Union[int, List[int]], dim: int = 0
+    ) -> List[TensorDictBase]:
+        """Splits each tensor in the TensorDict with the specified size in the given dimension, like `torch.split`.
+        Returns a list of TensorDict with the view of split chunks of tensor.
+
+        The list of TensorDict maintains the original order of the tensor chunks.
+
+        Args:
+            split_size (int) or (list(int)): size of a single chunk or list of sizes for each chunk
```

Comment:
`split_size (int of List[int]): ...`

Reply:
I'll update the type format of args `split_size(int or List(int))`.